src/storage/meta/entry/db_entry.cppm

```diff
@@ -61,7 +61,7 @@ public:
 
     static UniquePtr<DBEntry> Deserialize(const nlohmann::json &db_entry_json, BufferManager *buffer_mgr);
 
-    virtual void MergeFrom(BaseEntry &other); // TODO: fix warning
+    virtual void MergeFrom(BaseEntry &other) override;
```

Comment:
Remove virtual and change overide to final

Reply:
Also consider adding 'final' after the derived class name.

Reply:
done